application/models/TemplateConfig.php

```diff
@@ -111,17 +111,22 @@ public function getTemplateURL()
 
         /**
          * Get the template for a given file. It checks if a file exist in the current template or in one of its mother templates
-         *
+         * Can return a 302 redirect (this is not really a throw …
+         * 
          * @param  string $sFile the  file to look for (must contain relative path, unless it's a view file)
          * @param TemplateConfig $oRTemplate template from which the recurrence should start
-         * @return TemplateConfig
-         * @throws Exception
+         * @param boolean $force file to be in template or mother template
+         * @return TemplateConfig|null|void
          */
-        public function getTemplateForFile($sFile, $oRTemplate)
+        public function getTemplateForFile($sFile, $oRTemplate,$force=true)
         {
             while (!file_exists($oRTemplate->path.$sFile) && !file_exists($oRTemplate->viewPath.$sFile) && !file_exists($oRTemplate->filesPath.$sFile)) {
                 $oMotherTemplate = $oRTemplate->oMotherTemplate;
                 if (!($oMotherTemplate instanceof TemplateConfiguration)) {
+                    if(!$force && Yii::app()->twigRenderer->getPathOfFile($sFile)) {
```

Comment:
Here, it's only valid for twig file, broke for other (js/css and «other») but currently use $force to false for twig file.

Some alternative solution : getTemplateForFile($sFile, $oRTemplate,$filtetype=false)

1. phpDoc : @param boolean|string search file type on other way, uninstall if false or not find
2. Find the way to search on package for css/js, and maybe picture
3. Find something for other file (unsure ? : when we have asset for survey ?)